src/antsibull_docs/cli/antsibull_docs.py

```diff
@@ -367,6 +377,31 @@ def parse_args(program_name: str, args: List[str]) -> argparse.Namespace:
     sphinx_init_parser.add_argument('--index-rst-source',
                                     help='Copy the provided file to rst/index.rst intead of'
                                     ' templating a default one.')
+    sphinx_init_parser.add_argument('--project', default='Ansible collections',
+                                    help='Sets the "project" value in the Sphinx configuration.')
+    sphinx_init_parser.add_argument('--copyright', default='Ansible contributors',
+                                    help='Sets the "copyright" value in the Sphinx configuration.')
```

Comment:
Coming in late on this - but  I wonder/worry about this one. I'm assuming this has no impact on what shows up on docs.ansible.com, right? This would only allow someone using this to put their own copyright up on their own docsite somewheres.

But then what happens when docs.ansible.com has that content as 'copyright ansible project contributors' but the collection owners docsite says 'copyright Me Myself and I'?